sp/src/createallprojects

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 pushd `dirname $0`
-devtools/bin/vpc /hl2 /episodic +everything /mksln everything
+devtools/bin/vpc /ez1 +everything /mksln everything
```

Comment:
This is not correct. Entropy : Zero has its own create projects script to generate EZ1 and EZ2 slns - I did this so that HL2 / Episodic and EZ1 / EZ2 could be compiled from the same workspace. You could argue that ez1 should be built by createallprojects, but then HL2 and Episodic should as well.

Reply:
I see. I rebased the fix, which now simply adds a new script for the Linux build and keep the rest as-is.